smoke-test/tests/cypress/cypress/e2e/domains/nested_domains.js

```diff
@@ -0,0 +1,49 @@
+const domainName = "CypressNestedDomain";
+const domainDescription = "CypressNestedDomainDescription";
+
+describe("nested domains test", () => {
+
+    it("create a domain, move under parent, remove domain", () => {
+        //create a new domain without a parent
+        cy.loginWithCredentials();
+        cy.goToDomainList();
+        cy.clickOptionWithText("New Domain");
+        cy.waitTextVisible("Create New Domain");
+        cy.get('[data-testid="create-domain-name"]').click().type(domainName);
+        cy.get('#description').click().type(domainDescription);
+        cy.get('[data-testid="create-domain-button"]').click()
+        cy.waitTextVisible(domainName);
+        //ensure the new domain has no parent in the navigation sidebar
+        cy.waitTextVisible(domainDescription);
+        //move a domain from the root level to be under a parent domain
+        cy.clickOptionWithText(domainName);
+        cy.openThreeDotDropdown();
+        cy.clickOptionWithText("Move");
+        cy.get('[role="dialog"]').contains("Marketing").click({force: true});
+        cy.get('[role="dialog"]').contains("Marketing").should("be.visible");
+        cy.get("button").contains("Move").click().wait(5000);
+        //ensure domain is no longer on the sidebar navigator at the top level but shows up under the parent
+        cy.goToDomainList();
+        cy.ensureTextNotPresent(domainName);
+        cy.ensureTextNotPresent(domainDescription);
+        cy.waitTextVisible("1 sub-domain");
+        //move a domain from under a parent domain to the root level
+        cy.contains("Marketing").prev().click();
```

Comment:
is this clicking the arrow to expand the children of marketing? could we add a test ID on that button instead?

Reply:
yes, I've added data-testid for domain item. Tried to add to the expand button itself, but clicking on it doesn't do anything so I've kept using .prev() function with data-testid for domain item instead.